mapbox/dr-ui
# Control the Feedback textarea by the state, not the prop

This PR fixes a bug where the cursor in the textarea in Feedback jumps around. This is happening because the textarea is not properly controlled and so when a user clicks back into the textarea, React doesn't know where to place the cursor. We fix it by using the state to control the textarea, rather than the prop. 

Since `FeedbackTextarea` is lift state up to the parent component and the parent component does not make changes to the value, we should use the state to control the component.

Fixes #476 

## How to test

1. Run `npm start`
2. Visit /Feedback
3. Click through to any Feedback category
4. Enter text into the textarea.
5. Click anywhere on the page outside the textarea.
6. Click back into textarea as if you want to edit the start of your text. The cursor will **NOT** jump to the end of the text.
7. Submit feedback and make sure it comes through Sentry and #bots as expected.


## QA checklist




Open the test cases app locally on:


## Before merge
